browser/brave_rewards/rewards_service_factory.cc

```diff
@@ -124,10 +129,20 @@ KeyedService* RewardsServiceFactory::BuildServiceInstanceFor(
                         std::move(notification_observer));
   return rewards_service.release();
 #else
-  return NULL;
+  return nullptr;
 #endif
 }
 
+content::BrowserContext* RewardsServiceFactory::GetBrowserContextToUse(
+    content::BrowserContext* context) const {
+  if (context->IsOffTheRecord() || brave::IsTorProfile(context)) {
```

Comment:
This would never be reached when we have
https://github.com/brave/brave-core/pull/6418/files#diff-2e0dc7f5d8229da86d35994326ec274eR69
so why do we need this check here? Even if we allow tor profile to access reward service, it should be using original profile.

Reply:
I think we only need GetForProfile change and no need to overwrite GetBrowserContextToUse here

Reply:
same thing applies to https://github.com/brave/brave-core/pull/6332/files#diff-3639f92629488e9f547d8ff7ead4a5f7R74

Reply:
Fixed